src/Stratis.Bitcoin/P2P/Protocol/Message.cs

```diff
@@ -194,8 +169,7 @@ public static Message ReadNext(Stream stream, Network network, ProtocolVersion v
             if (!network.ReadMagic(stream, cancellationToken, true))
                 throw new FormatException("Magic incorrect, the message comes from another network");
 
-            Message message = new Message();
-            message.buffer = buffer;
+            Message message = new Message(payloadProvider);
```

Comment:
`var message`

Reply:
@dangershony Just fix this and we are good to go!

Reply:
I don't mind keeping it as type (@Aprogiena got to me I guess)

Reply:
I don't mind using var here. And François is right that according to the policy it should be here instead of the type.